tests/test_sourceSelector.py

```diff
@@ -82,6 +83,23 @@ def testFlags(self):
         self.config.flags.bad = ["badFlag"]
         self.check([False, False, False, True])
 
+    def testSignalToNoise(self):
+        low = self.catalog.addNew()
+        low.set("other_flux", 1.0)
+        low.set("other_fluxSigma", 1.0)
+        good = self.catalog.addNew()
+        good.set("other_flux", 1.0)
+        good.set("other_fluxSigma", 0.1)
+        high = self.catalog.addNew()
+        high.set("other_flux", 1.0)
+        high.set("other_fluxSigma", 0.001)
+        self.config.doSignalToNoise = True
+        self.config.signalToNoise.fluxField = "other_flux"
+        self.config.signalToNoise.errField = "other_fluxSigma"
```

Comment:
I think @parejkoj is trying to homogenize the use of err vs. Sigma in the stack (RFC-333).

Reply:
Sadly, none of that has been implemented yet, so `fluxSigma` will remain.